pkgs/tools/misc/sd-mux-ctrl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, cmake, pkgconfig, libftdi1, popt}:
+
+stdenv.mkDerivation rec {
+  pname = "sd-mux-ctrl";
```

Comment:
From the manual, [https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming](18.2. Package naming)

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

The `pname` should thus be `"sd-mux-ctrl-unstable"`.

Reply:
Thank you for the context, Ill keep that in mind for the future